File: exercise/java/day14/src/main/java/children/dtos/Child.java
package children.dtos;

import lombok.Data;

import java.time.LocalDate;

@Data
public class Child {
    private String id;
    private String firstName;
    private String middleName;
    private String lastName;
    private String birthCity;
    private LocalDate birthDate;
    private Gender gender;
    private Address address;
}

File: exercise/php/day14/src/Children/Dtos/Child.php
<?php

namespace Children\DTOs;

class Child {
    public string $id;
    public ?string $firstName;
    public ?string $middleName;
    public ?string $lastName;
    public ?string $birthCity;
    public string $birthDate;
    public Gender $gender;
    public ?Address $address;
}

